### DISCUSSION TITLE
Upgrade issue for truffle

### DISCUSSION THREAD

**Discussion by keshari1arya** [ConsenSys-archive/truffle#4638] [General Q&A]
Hi,
I am trying to upgrade truffle to the latest version but after uninstalling and installing I am getting bellow error. 

truffle init
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'C:\Users\<username>\AppData\Roaming\npm\node_modules\truffle\build\cli.bundled.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

npm -v 8.3.2
node -v v16.13.0

commands i used:
npm uninstall truffle -g
npm install truffle -g

**Comment by apdan510** [comment]
@keshari1arya try:
npm uninstall truffle -g --force
if that does not work:
Try to remove /truffle directories from any or all of your node_modules/ folders.
